modules/index/src/test/java/org/apache/ignite/internal/index/ChangeIndexStatusTaskControllerTest.java

```diff
@@ -76,15 +76,15 @@ public class IndexTaskManagerTest extends BaseIgniteAbstractTest {
     @Mock
     private ChangeIndexStatusTaskScheduler changeIndexStatusTaskScheduler;
 
-    private @Nullable IndexTaskManager taskManager;
+    private @Nullable ChangeIndexStatusTaskController taskManager;
```

Comment:
I think it should be renamed to `taskController` or `controller`.

Reply:
Fixed